gamersnet_backend/persistence/users.js

```diff
@@ -37,4 +45,17 @@ async function updateUserPassword(id, hashedPassword) {
     );
 }
 
-module.exports = {addUser, getUserByUsername, updateUserPassword};
\ No newline at end of file
+async function upsertDetails(id, details) {
```

Comment:
may wanna change the name. It isn't exactly upserting. It won't update the document if it can't find one(and this is what we want, we don't want to create a new document). To upsert, the upsert option is set to true.

Reply:
Thats a good point, this seems to be another case of where I name a function something completely different than what it actually does. This will be changed to `updateDetails`